Domaineinstellungen unter yrewrite

anveno opened the pull request:
Die Domaineinstellungen erscheinen dadurch nicht mehr unter "Tabellen", sondern als Reiter bei "Addons" > "YRewrite".

siehe auch #3

anveno committed:
Domaineinstellungen unter yrewrite

Die Domaineinstellungen erscheinen dadurch nicht mehr unter "Tabellen", sondern als Reiter bei "Addons" > "YRewrite".

siehe auch #3

skerbis commented:
Als Reiter in YRewrite hätten die Redakteur:innen keinen Zugriff mehr auf die Einstellungen. Das sollen sie bei mir aber haben.

skerbis reviewed (changes requested):
Als Reiter in YRewrite hätten die Redakteur:innen keinen Zugriff mehr auf die Einstellungen. Das sollen sie bei mir aber haben. Ein eigener Menüpunkt wäre da sicher ein Kompromiss. 

Meine Kunden sollen dort z.B. ihre Footer selbst pflegen oder auch mal das Logo und Kontaktdaten tauschen. Durch ein Verschieben in YRewrite hätten Sie keinen Zugriff mehr drauf weil dieses Admins vorbehalten ist.

anveno commented:
Verständlicher Einwand.

Alternativ zum eigenen Menüpunkt könnte man es auch in der install.php wieder auf "hidden": "0" setzen.
Und kann dann nach der Installation optional im Table Manager "In Navigation versteckt" aktivieren.

Aber eine eigene Seite ist vermutlich sinnvoller; evtl. "YRewrite Settings", so das es direkt unter "YRewrite" unter Addons erscheint?

AWqxKAWERbXo commented:
Ansonsten nimmste halt meins 😉
https://github.com/alexplusde/yrewrite_metainfo/blob/25b95550c0325955d3e8dfaf8419562e415ef315/package.yml#L15-L17

Spaß beiseite, YRewrite soll ein eigenes Recht für den Hauptmenü-Punkt vergeben, dann kann man diesen aktivieren ohne den Rest: https://github.com/yakamara/redaxo_yrewrite/blob/6123eedde97ac2ab788633ead92b4ca3f5d0edc2/package.yml#L7-L16

anveno committed:
Ansatz für Mehrsprachigkeit

Neues Feld "domain_lang".
"validate unique" um "domain_lang" erweitert.

a) yrewrite_domain_settings::getValue('meineintrag');
holt Value der Start-Sprache (rex_clang::getStartId).

b) yrewrite_domain_settings::getValue('meineintrag', true);
holt Value der aktuellen Sprache (rex_clang::getCurrentId()). Sofern leer Fallback auf a)

anveno commented:
@alxndr-w 
Dein Addon scheint generell etwas aktueller zu sein. Allerdings ist mir das mit den vorbelgten Feldern und zusätzlichen Tabellen schon fast zu viel. So ein Zwischending wäre wahrscheinlich super, d.h. das man das optional importieren kann.

Nun denn. Habe hier gerade einen falschen zweiten Commit reingetüdelt... wie ziehe ich den Pull Request zurück?
Via "Close with comment" oder an welcher Stelle? Hab´s immer noch nicht so mit Git :/

skerbis commented:
Hey ich wollte das nicht kaputt reden

anveno commented:
> Hey ich wollte das nicht kaputt reden

@skerbis Alles gut, habe es nur geschlossen da ich hier zwei verschiedene Sachen im Git vermischt habe - ich maches es nochmal sauber mit Branches. Muss mich hier erst reinfummeln.

AWqxKAWERbXo commented:
Gerne erweitere ich euch noch dieses Domainauswahl-Feld für YForm um eine Sprachauswahl - beim nächsten mehrsprachigen Kunden. 

https://github.com/alexplusde/yform_field/blob/main/lib/yform/value/domain.php
